Content.Server/GameTicking/Rules/PiratesRuleSystem.cs

```diff
@@ -218,6 +220,13 @@ private void OnPlayerSpawningEvent(RulePlayerSpawningEvent ev)
 
                 pirates.Pirates.Add(newMind);
 
+                // Notificate every player about a pirate antagonist role with sound
+                _audio.PlayGlobal(
+                    pirates.PirateAlertSound,
+                    Filter.SinglePlayer(session),
```

Comment:
This method is explicitly done as an admin option to spawn a pirate.
It will not trigger any logic done at the start of the round at OnPlayerSpawningEvent.
BUT you are right, those methods (SetOutfitCommand.SetOutfit and StationSpawningSystem.EquipStartingGear) do the same stuff, but there are different checks cause of the lack of components at the start of the execution.
I believe that this is a task for refactoring that is out of the scope of this PR.